app/code/Magento/CatalogImportExport/Model/Import/Product/CategoryProcessor.php

```diff
@@ -204,4 +221,21 @@ public function getCategoryById($categoryId)
     {
         return isset($this->categoriesCache[$categoryId]) ? $this->categoriesCache[$categoryId] : null;
     }
+
+    /**
+     * Standardize a string by lowercase-ing it
+     *
+     * @param string $string
+     * @return string
+     */
+    protected function standardizeString(string $string)
```

Comment:
Removing the `function_exists` check would eliminate the need in the additional method.